azext_edge/edge/vendor/clients/deviceregistrymgmt/_vendor.py

```diff
@@ -0,0 +1,20 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from typing import List, cast
+
+
+def _format_url_section(template, **kwargs):
```

Comment:
how the heck did you manage to generate this file? I wasnt able to but it could also be a skill issue...